executor/Cargo.toml

```diff
@@ -5,31 +5,37 @@ authors = ["Centrality Developers <support@centrality.ai>"]
 edition = "2018"
 
 [dependencies]
-trie-root = "0.15.2"
+# third-party dependencies
 codec = { package = "parity-scale-codec", version = "1.0.0" }
-runtime-io = { package = "sp-io", git = "https://github.com/plugblockchain/plug-blockchain", branch = "1.0.0-rc2" }
-state-machine = { package = "sp-state-machine", git = "https://github.com/plugblockchain/plug-blockchain", branch = "1.0.0-rc2" }
-sc-executor = { git = "https://github.com/plugblockchain/plug-blockchain", branch = "1.0.0-rc2" }
-primitives = { package = "sp-core", git = "https://github.com/plugblockchain/plug-blockchain", branch = "1.0.0-rc2" }
-trie = { package = "sp-trie", git = "https://github.com/plugblockchain/plug-blockchain", branch = "1.0.0-rc2" }
+trie-root = "0.15.2"
+
+# internal dependencies
 cennznet-primitives = { path = "../primitives" }
 cennznet-runtime = { path = "../runtime" }
 
+#frame dependencies
+# runtime-io = { package = "sp-io", git = "https://github.com/plugblockchain/plug-blockchain", branch = "1.0.0-rc2" }
```

Comment:
Why some deps are commented?

Reply:
Oh, we forgot it, these should be removed, done